### PR TITLE
feat(sdk): Fix for tests for python sdk and environment variables for…

### PR DESCRIPTION
**Pull request by avinashpipalwa**
… examples

Added changes for tests for python sdk.
Fixed the username variables to get the username serial from the environment variables.

### CODE REVIEW

**Comment by ksreeramoj** [comment]
Can you remove the commented lines.

File: tests/test_linked_pa_templates.py [hunk 1]
```diff
@@ -80,15 +80,16 @@ def test_c_update_linked_pa_template(self):
 
         self.assertEqual(response[1], 200, "Response should be 200 - Success")
         self.assertEqual(type(response[0].data), dict, "Response should be of Dictionary type.")
-        self.assertEqual(type(response[0].data[template_id]),
-            LinkedPATemplateSummary, "Response should be of LinkedPATemplateSummary type.")
+        self.assertEqual(type(response[0].data),
+            dict, "Response should be of LinkedPATemplateSummary type.")
```
Comment (Ramanth) [inline tests/test_linked_pa_templates.py:84]: looks like you are checking for dict type twice and the validation message is different

**Review by Ramanth — APPROVED** [review]
left minor comment

File: tests/test_linked_pa_templates.py (hunk 1 above)
Comment (avinashpipalwa) [reply tests/test_linked_pa_templates.py:84]: will change the validation message to dict type, missed this.

**Review by ksreeramoj — APPROVED** [review]

**Review by Ramanth — APPROVED** [review]

**Pull request merged**